frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -59,3 +60,23 @@ export const getOCSVersion = (items: FirehoseResult): string => {
   );
   return _.get(operator, 'status.installedCSV');
 };
+
+export const getSCAvailablePVs = (sc: string) =>
+  k8sList(PersistentVolumeModel)
+    .catch(() => [])
+    .then((pvsData: K8sResourceKind[]) => {
+      return pvsData.filter(
+        (pv) =>
+          _.isEqual(_.get(pv, 'spec.storageClassName'), sc) &&
+          _.isEqual(_.get(pv, 'status.phase'), 'Available'),
+      );
+    });
+
+export const calcPVsCapacity = (pvs: K8sResourceKind[]) =>
+  _.reduce(
+    pvs,
+    function(sum, pv) {
+      return sum + Number(convertToBaseValue(_.get(pv, 'spec.capacity.storage', '0')));
+    },
+    0,
```

Comment:
```suggestion
  _.reduce(
    pvs,
    (sum, pv) =>  {
    const storage = Number(convertToBaseValue(pv.spec.capacity?.storage) ?? '0');
    return sum + storage;
    }, 0 );
```
Avoid using functions. Prefer arrow functions `=>`